packages/core/src/Button/Button.tsx

```diff
@@ -110,6 +110,18 @@ const variations = {
       box-shadow: 0 0 0 2px ${getPaletteColor('dark')};
     }
   `,
+  white: css`
```

Comment:
should we call this `lightestFill` instead so it's naming is more dark mode compatible?